src/components/input-stuff.js

```diff
@@ -1,30 +1,25 @@
 import React from 'react';
-import {connect} from 'react-redux';
 
-import {addBrew} from '../database/brew';
-
-var testEquip = ["Chemex", "Clever", "French Press"];
 
 const FancyInput = (props) => {
   return (
- 	    <label className="input-label"> {props.label} 
-      <input name={props.name} type="text" />
-      </label> 
+    <label className="input-label"> {props.label}
+      <input name={props.name} handleChange={props.handleChange} type="text" />
```

Comment:
this one does need to be `onChange={props.handleChange}`

the use of `handle` vs `on` is a react-specific one. here i wanted to use it just to avoid the case where you have `onChange={props.onChange}` which always hurts my head, but the idea is that something named `handleWhatever` is actually a function that modifies state somewhere. a function or prop named `onWhatever` is used to advertise "whatever you pass to this prop will be called on whatever event it is bound to".

that is, onWhatever is a public-facing api for a component/input that is called on the relevant event whereas handleWhatever is the function that usually goes there. Ideally you have:

```js
handleTextInputChange = evt => this.setState({someStateKey: evt.target.value})
render() {
  return <input onChange={this.handleTextInputChange} />
}
```